jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -149,6 +150,18 @@ public JavaDockerContextGenerator setBaseImage(String baseImage) {
     return this;
   }
 
+  /**
+   * Sets the entrypoint to be used as the {@code ENTRYPOINT}. If not empty, then overrides the
+   * {@link #setJvmFlags(List) jvmFlags} and {@link #setMainClass(String) mainclass}.
```

Comment:
I think if we have this method, we should just remove the `setJvmFlags` and `setMainClass` methods and delegate the call of `JavaEntrypointConstructor#makeEntrypoint` to the user of `JavaDockerContextGenerator`.

Reply:
Makes sense.

Reply:
There's not much specific to Java in `JavaDockerContextGenerator` now!